File: Assignment__33/Question4.c
#include<stdio.h>
#include<stdbool.h>

bool checkDigit(char *str)
{
    while(*str != '\0')
    {
        if((*str >= 97 && *str <= 122))
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

int main()
{
    char Arr[50] = "\0";
    bool bRet = false;

    printf("Enter the String : \n");
    scanf("%[^\n]s",Arr);

    bRet = checkDigit(Arr);

    if(bRet == true)
    {
        printf("It is in small case.");
    }
    else
    {
        printf("It is not in small case.");
    }

    return 0;
}

File: Assignment__33/Question4.java
import java.util.*;

class Alphabet
{
    public boolean Check(char chh)
    {
        if((chh >= 97 && chh <= 122))
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}
class Question4
{
    public static void main(String A[])
    {
        boolean bRet = false;

        Scanner scanobj = new Scanner(System.in);
        System.out.println("Enter the input : ");

        char ch = scanobj.next().charAt(0);               //Need to remember

        Alphabet aobj = new Alphabet();

        bRet = aobj.Check(ch);

        if(bRet == true)
        {
            System.out.println("It is in small case");
        }
        else
        {
            System.out.println("It is not in small case");
        }

    }
}